e2e/tests/mcp-setup-wizard.spec.ts

```
@@ -146,7 +146,7 @@ test.describe('MCP Management', () => {
     test('create new gateway radio expands embedded form', { tag: '@nightly' }, async ({ page }) => {
       await spaNavigate(page, '/kuadrant/mcp/setup-wizard');
 
-      await expect(page.getByLabel('Choose a Gateway')).toBeChecked({ timeout: 15_000 });
+      await expect(page.getByLabel('Choose an existing Gateway')).toBeChecked({ timeout: 15_000 });
```

ESLint

[error] Replace `······await·expect(page.getByLabel('Choose·an·existing·Gateway')).toBeChecked({·timeout:·15_000` with `········await·expect(page.getByLabel('Choose·an·existing·Gateway')).toBeChecked({⏎··········timeout:·15_000,⏎·······`

(prettier/prettier)